editors/tinymce5/fontawesome/plugin.min.js

```diff
@@ -0,0 +1,4781 @@
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Bitbucket",
+			"id": "bitbucket",
+			"unicode": "f171",
+			"created": 3.2,
+			"filter": ["git"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Bitbucket Square",
+			"id": "bitbucket-square",
+			"unicode": "f172",
+			"created": 3.2,
+			"filter": ["git"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Tumblr",
+			"id": "tumblr",
+			"unicode": "f173",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Tumblr Square",
+			"id": "tumblr-square",
+			"unicode": "f174",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Long Arrow Down",
+			"id": "long-arrow-down",
+			"unicode": "f175",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Up",
+			"id": "long-arrow-up",
+			"unicode": "f176",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Left",
+			"id": "long-arrow-left",
+			"unicode": "f177",
+			"created": 3.2,
+			"filter": ["previous", "back"],
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Right",
+			"id": "long-arrow-right",
+			"unicode": "f178",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Apple",
+			"id": "apple",
+			"unicode": "f179",
+			"created": 3.2,
+			"filter": ["osx", "food"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Windows",
+			"id": "windows",
+			"unicode": "f17a",
+			"created": 3.2,
+			"filter": ["microsoft"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Android",
+			"id": "android",
+			"unicode": "f17b",
+			"created": 3.2,
+			"filter": ["robot"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Linux",
+			"id": "linux",
+			"unicode": "f17c",
+			"created": 3.2,
+			"filter": ["tux"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Dribbble",
+			"id": "dribbble",
+			"unicode": "f17d",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Skype",
+			"id": "skype",
+			"unicode": "f17e",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Foursquare",
+			"id": "foursquare",
+			"unicode": "f180",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Trello",
+			"id": "trello",
+			"unicode": "f181",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Female",
+			"id": "female",
+			"unicode": "f182",
+			"created": 3.2,
+			"filter": ["woman", "user", "person", "profile"],
+			"categories": ["Web Application Icons"]
+		}, {
+			"name": "Male",
+			"id": "male",
+			"unicode": "f183",
+			"created": 3.2,
+			"filter": ["man", "user", "person", "profile"],
+			"categories": ["Web Application Icons"]
+		}, {
+			"name": "Gratipay (Gittip)",
+			"id": "gratipay",
+			"unicode": "f184",
+			"created": 3.2,
+			"aliases": ["gittip"],
+			"filter": ["heart", "like", "favorite", "love"],
+			"categories": ["Brand Icons"]
+
```

Comment:
@cmb69: mir ist grad auf die Schnelle nicht die jQuery freie Variante eingefallen. Vielleicht hast Du sie aus dem Stegreif.

Reply:
For modern browsers one could use:
````
document.querySelector('.mce-fontawesome-panel-accordion').closest('[role="dialog"').classList.add('mce-fontawesome-panel');
````
That won't work for IE, though, but it shouldn't be hard to work around that. BTW, do you know which browsers are supported by TinyMCE5 ([4.6 already dropped IE ≤ 10](https://www.tiny.cloud/docs/general-configuration-guide/system-requirements/))?

Reply:
Es wäre tatsächlich nicht schwer einen Workaround für IE 11 einzubauen, aber das ist wohl gar nicht nötig, siehe weiter unten.